lms/static/js/groups/views/course_cohort_settings_notification.js

```diff
@@ -0,0 +1,35 @@
+var edx = edx || {};
+
+(function($, _, Backbone, gettext) {
+    'use strict';
+
+    edx.groups = edx.groups || {};
+
+    edx.groups.CourseCohortSettingsNotificationView = Backbone.View.extend({
+        initialize: function(options) {
+            this.template = _.template($('#cohort-state-tpl').text());
+            this.cohortEnabled = options.cohortEnabled;
+        },
+
+        render: function() {
+            this.$el.html(this.template({}));
+            this.showCohortStateMessage();
+            return this;
+        },
+
+        showCohortStateMessage: function () {
+            var actionToggleMessage = this.$('.action-toggle-message');
+
+            // The following lines are necessary to re-trigger the CSS animation on span.action-toggle-message
+            actionToggleMessage.removeClass('is-fleeting');
```

Comment:
I see that these lines are copied from Student Notes. Can you move them into a helper to make it clearer what it is doing.